backend/common/config_dev.go

```diff
@@ -0,0 +1,7 @@
+//go:build !release
+
+package common
+
+func IsDev() bool {
```

Comment:
Can we just use the `profile.Mode`?

Reply:
too troublesome, pass profiles around....